### ISSUE TITLE
Module not found: Error: Can't resolve 'core-js/modules/web.dom-collections.iterator'

### ISSUE THREAD

**Issue by altryne** [nuxt-community/storybook#59]
### Version
@nuxtjs/storybook: v1.0.1
nuxt: ex: v2.14.0

### Steps to reproduce
Install per docs
run `npx nuxt storybook` 

### What is Expected?
Per docs, if I use nuxt 2.14 then I don't need to install core-js additionally

### What is actually happening?
Core JS complains and running `yarn add --dev core-js@3 @babel/runtime-corejs3` fixes this issue

**Comment by farnabaz** [comment]
Pelease update both `nuxt` and `@nuxtjs/storybook` to latest version. In the latest version this problem resolves automatically

**Comment by pa-bouly** [comment]
I think it's coming from storybook, see [Core-JS dependency errors](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#core-js-dependency-errors)

**Comment by Jursdotme** [comment]
```
"nuxt": "^2.14.3",
"@nuxtjs/storybook": "^2.1.0",
```

I still have the error and get multiple variations of this:

```
ERROR in ./store/index.js
Module not found: Error: Can't resolve 'core-js/modules/es.object.to-string' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\store'
 @ ./store/index.js 9:0-45 17:0-45
 @ ./.nuxt-storybook/store.js
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/index.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/index.js 17:0-36
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/router.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/router.js 2:0-36
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/router.scrollBehavior.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/router.scrollBehavior.js 4:0-36
 @ ./.nuxt-storybook/router.js
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/utils.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/utils.js 25:0-36
 @ ./.nuxt-storybook/mixins/fetch.client.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/axios.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/axios.js 19:0-36
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined
```

**Comment by Jursdotme** [comment]
> Pelease update both `nuxt` and `@nuxtjs/storybook` to latest version. In the latest version this problem resolves automatically

```
"nuxt": "^2.14.3",
"@nuxtjs/storybook": "^2.1.0",
```
These are the latest releases of both packages at the time of writing this.
I still have the error and get multiple variations of this:

```
ERROR in ./store/index.js
Module not found: Error: Can't resolve 'core-js/modules/es.object.to-string' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\store'
 @ ./store/index.js 9:0-45 17:0-45
 @ ./.nuxt-storybook/store.js
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/index.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/index.js 17:0-36
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/router.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/router.js 2:0-36
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/router.scrollBehavior.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/router.scrollBehavior.js 4:0-36
 @ ./.nuxt-storybook/router.js
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/utils.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/utils.js 25:0-36
 @ ./.nuxt-storybook/mixins/fetch.client.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined

ERROR in ./.nuxt-storybook/axios.js
Module not found: Error: Can't resolve 'core-js/modules/es.promise' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook'
 @ ./.nuxt-storybook/axios.js 19:0-36
 @ ./.nuxt-storybook/index.js
 @ ./.nuxt-storybook/storybook/entry.js
 @ ./.nuxt-storybook/storybook/storybook-init-framework-entry.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined
```

**Comment by Jursdotme** [comment]
After installing `core-js@3` i get the following error:

```
PS C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt> npx nuxt storybook
i Using Tailwind CSS from ~/assets\css\tailwind.css                                                                                                                                                                                                                                    nuxt:tailwindcss 22:15:53
i Merging Tailwind config from ~/tailwind.config.js                                                                                                                                                                                                                                    nuxt:tailwindcss 22:15:53
You are using a low quality icon, use icon png. See https://pwa.nuxtjs.org/icon/
i Preparing project for development                                                                                                                                                                                                                                                                     22:15:54
i Initial build may take a while                                                                                                                                                                                                                                                                        22:15:54
√ Builder initialized                                                                                                                                                                                                                                                                                   22:15:55
√ Nuxt files generated                                                                                                                                                                                                                                                                                  22:15:55
i Waiting for file changes                                                                                                                                                                                                                                                                              22:15:55
info @nuxtjs/storybook v2.1.0
info
info => Loading static files from: C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\static .
info => Loading presets
info => Loading presets
info => Loading config/preview file in "C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook\storybook".
info => Loading config/preview file in "C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook\storybook".
WARN We found no files matching any of the following globs:
WARN
WARN ..../components/**/*.stories.@(ts|js)
WARN
WARN Maybe your glob was invalid?
WARN see: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs
info => Using default Webpack setup.

× Client
  Compiled with some errors in 24.25s


 ERROR  Failed to compile with 1 errors                                                                                                                                                                                                                                                 friendly-errors 22:16:20

This dependency was not found:                                                                                                                                                                                                                                                          friendly-errors 22:16:20
                                                                                                                                                                                                                                                                                        friendly-errors 22:16:20
* ..../components in ./.nuxt-storybook/storybook/generated-stories-entry.js                                                                                                                                                                                                             friendly-errors 22:16:20
                                                                                                                                                                                                                                                                                        friendly-errors 22:16:20
To install it, you can run: npm install --save ..../components                                                                                                                                                                                                                          friendly-errors 22:16:20
webpack built 61034e25cd5ac7b40022 in 24255ms
× ｢wdm｣: Hash: 61034e25cd5ac7b40022
Version: webpack 4.44.1
Time: 24255ms
Built at: 2020-08-17 10:16:20 PM
                                               Asset       Size             Chunks                                Chunk Names
                       icons/icon_120x120.5f6a36.png   4.68 KiB                     [emitted]
                       icons/icon_144x144.5f6a36.png    5.8 KiB                     [emitted]
                       icons/icon_152x152.5f6a36.png    6.1 KiB                     [emitted]
                       icons/icon_192x192.5f6a36.png   7.83 KiB                     [emitted]
                       icons/icon_384x384.5f6a36.png   18.1 KiB                     [emitted]
                       icons/icon_512x512.5f6a36.png     20 KiB                     [emitted]
                         icons/icon_64x64.5f6a36.png   2.35 KiB                     [emitted]
              icons/splash_ipad_1536x2048.5f6a36.png    157 KiB                     [emitted]
         icons/splash_ipadpro10_1668x2224.5f6a36.png    194 KiB                     [emitted]
         icons/splash_ipadpro12_2048x2732.5f6a36.png    274 KiB                     [emitted]              [big]
          icons/splash_ipadpro9_1536x2048.5f6a36.png    157 KiB                     [emitted]
             icons/splash_iphone6_50x1334.5f6a36.png   2.33 KiB                     [emitted]
        icons/splash_iphoneplus_1080x1920.5f6a36.png   99.3 KiB                     [emitted]
           icons/splash_iphonese_640x1136.5f6a36.png   41.7 KiB                     [emitted]
           icons/splash_iphonex_1125x2436.5f6a36.png    108 KiB                     [emitted]
           icons/splash_iphonexr_828x1792.5f6a36.png     64 KiB                     [emitted]
       icons/splash_iphonexsmax_1242x2688.5f6a36.png    127 KiB                     [emitted]
                                         iframe.html   2.92 KiB                     [emitted]
                 main.61034e25cd5ac7b40022.bundle.js   5.99 MiB               main  [emitted] [immutable]  [big]  main
             main.61034e25cd5ac7b40022.bundle.js.map   5.99 MiB               main  [emitted] [dev]               main
                              manifest.b81a7629.json  155 bytes                     [emitted]
    pages/_slug/index.61034e25cd5ac7b40022.bundle.js   15.7 KiB  pages/_slug/index  [emitted] [immutable]         pages/_slug/index
pages/_slug/index.61034e25cd5ac7b40022.bundle.js.map   5.08 KiB  pages/_slug/index  [emitted] [dev]               pages/_slug/index
     pages/blog/_slug.61034e25cd5ac7b40022.bundle.js   12.9 KiB   pages/blog/_slug  [emitted] [immutable]         pages/blog/_slug
 pages/blog/_slug.61034e25cd5ac7b40022.bundle.js.map   5.73 KiB   pages/blog/_slug  [emitted] [dev]               pages/blog/_slug
     pages/blog/index.61034e25cd5ac7b40022.bundle.js   15.1 KiB   pages/blog/index  [emitted] [immutable]         pages/blog/index
 pages/blog/index.61034e25cd5ac7b40022.bundle.js.map   10.1 KiB   pages/blog/index  [emitted] [dev]               pages/blog/index
        pages/contact.61034e25cd5ac7b40022.bundle.js   22.3 KiB      pages/contact  [emitted] [immutable]         pages/contact
    pages/contact.61034e25cd5ac7b40022.bundle.js.map   19.9 KiB      pages/contact  [emitted] [dev]               pages/contact
      pages/employees.61034e25cd5ac7b40022.bundle.js   16.4 KiB    pages/employees  [emitted] [immutable]         pages/employees
  pages/employees.61034e25cd5ac7b40022.bundle.js.map   11.5 KiB    pages/employees  [emitted] [dev]               pages/employees
          pages/index.61034e25cd5ac7b40022.bundle.js   9.74 KiB        pages/index  [emitted] [immutable]         pages/index
      pages/index.61034e25cd5ac7b40022.bundle.js.map   4.03 KiB        pages/index  [emitted] [dev]               pages/index
         runtime~main.61034e25cd5ac7b40022.bundle.js   36.5 KiB       runtime~main  [emitted] [immutable]         runtime~main
     runtime~main.61034e25cd5ac7b40022.bundle.js.map   37.9 KiB       runtime~main  [emitted] [dev]               runtime~main
         vendors~main.61034e25cd5ac7b40022.bundle.js   4.83 MiB       vendors~main  [emitted] [immutable]  [big]  vendors~main
     vendors~main.61034e25cd5ac7b40022.bundle.js.map   4.39 MiB       vendors~main  [emitted] [dev]               vendors~main
Entrypoint main [big] = runtime~main.61034e25cd5ac7b40022.bundle.js runtime~main.61034e25cd5ac7b40022.bundle.js.map vendors~main.61034e25cd5ac7b40022.bundle.js vendors~main.61034e25cd5ac7b40022.bundle.js.map main.61034e25cd5ac7b40022.bundle.js main.61034e25cd5ac7b40022.bundle.js.map
[0] multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined 136 bytes {main} [built]
[./.nuxt-storybook/client.js] 41.1 KiB {main} [built]
[./.nuxt-storybook/components/nuxt-link.client.js] 6.75 KiB {main} [built]
[./.nuxt-storybook/index.js] 16.8 KiB {main} [built]
[./.nuxt-storybook/middleware.js] 47 bytes {main} [built]
[./.nuxt-storybook/mixins/fetch.client.js] 3.68 KiB {main} [built]
[./.nuxt-storybook/storybook/generated-stories-entry.js] 263 bytes {main} [built]
[./.nuxt-storybook/storybook/preview.js-generated-config-entry.js] 3.26 KiB {main} [built]
[./.nuxt-storybook/storybook/storybook-init-framework-entry.js] 213 bytes {main} [built]
[./node_modules/@nuxt/components/lib/installComponents.js] 1010 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js] 2.31 KiB {vendors~main} [built]
[./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js] 2.31 KiB {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/common/polyfills.js] 120 bytes {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/preview/globals.js] 93 bytes {vendors~main} [built]
[./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined] (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined 7.68 KiB {vendors~main} [built]
    + 1904 hidden modules

ERROR in ./.nuxt-storybook/storybook/generated-stories-entry.js
Module not found: Error: Can't resolve '..../components' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook\storybook'
 @ ./.nuxt-storybook/storybook/generated-stories-entry.js 5:27-165
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/clien             icons/splash_iphone6_50x1334.5f6a36.png   2.33 KiB                     [emitted]
        icons/splash_iphoneplus_1080x1920.5f6a36.png   99.3 KiB                     [emitted]
           icons/splash_iphonese_640x1136.5f6a36.png   41.7 KiB                     [emitted]
           icons/splash_iphonex_1125x2436.5f6a36.png    108 KiB                     [emitted]
           icons/splash_iphonexr_828x1792.5f6a36.png     64 KiB                     [emitted]
       icons/splash_iphonexsmax_1242x2688.5f6a36.png    127 KiB                     [emitted]
                                         iframe.html   2.92 KiB                     [emitted]
                 main.61034e25cd5ac7b40022.bundle.js   5.99 MiB               main  [emitted] [immutable]  [big]  main
             main.61034e25cd5ac7b40022.bundle.js.map   5.99 MiB               main  [emitted] [dev]               main
                              manifest.b81a7629.json  155 bytes                     [emitted]
    pages/_slug/index.61034e25cd5ac7b40022.bundle.js   15.7 KiB  pages/_slug/index  [emitted] [immutable]         pages/_slug/index
pages/_slug/index.61034e25cd5ac7b40022.bundle.js.map   5.08 KiB  pages/_slug/index  [emitted] [dev]               pages/_slug/index
     pages/blog/_slug.61034e25cd5ac7b40022.bundle.js   12.9 KiB   pages/blog/_slug  [emitted] [immutable]         pages/blog/_slug
 pages/blog/_slug.61034e25cd5ac7b40022.bundle.js.map   5.73 KiB   pages/blog/_slug  [emitted] [dev]               pages/blog/_slug
     pages/blog/index.61034e25cd5ac7b40022.bundle.js   15.1 KiB   pages/blog/index  [emitted] [immutable]         pages/blog/index
 pages/blog/index.61034e25cd5ac7b40022.bundle.js.map   10.1 KiB   pages/blog/index  [emitted] [dev]               pages/blog/index
        pages/contact.61034e25cd5ac7b40022.bundle.js   22.3 KiB      pages/contact  [emitted] [immutable]         pages/contact
    pages/contact.61034e25cd5ac7b40022.bundle.js.map   19.9 KiB      pages/contact  [emitted] [dev]               pages/contact
      pages/employees.61034e25cd5ac7b40022.bundle.js   16.4 KiB    pages/employees  [emitted] [immutable]         pages/employees
  pages/employees.61034e25cd5ac7b40022.bundle.js.map   11.5 KiB    pages/employees  [emitted] [dev]               pages/employees
          pages/index.61034e25cd5ac7b40022.bundle.js   9.74 KiB        pages/index  [emitted] [immutable]         pages/index
      pages/index.61034e25cd5ac7b40022.bundle.js.map   4.03 KiB        pages/index  [emitted] [dev]               pages/index
         runtime~main.61034e25cd5ac7b40022.bundle.js   36.5 KiB       runtime~main  [emitted] [immutable]         runtime~main
     runtime~main.61034e25cd5ac7b40022.bundle.js.map   37.9 KiB       runtime~main  [emitted] [dev]               runtime~main
         vendors~main.61034e25cd5ac7b40022.bundle.js   4.83 MiB       vendors~main  [emitted] [immutable]  [big]  vendors~main
     vendors~main.61034e25cd5ac7b40022.bundle.js.map   4.39 MiB       vendors~main  [emitted] [dev]               vendors~main
Entrypoint main [big] = runtime~main.61034e25cd5ac7b40022.bundle.js runtime~main.61034e25cd5ac7b40022.bundle.js.map vendors~main.61034e25cd5ac7b40022.bundle.js vendors~main.61034e25cd5ac7b40022.bundle.js.map main.61034e25cd5ac7b40022.bundle.js main.61034e25cd5ac7b40022.bundle.js.map
[0] multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined 136 bytes {main} [built]
[./.nuxt-storybook/client.js] 41.1 KiB {main} [built]
[./.nuxt-storybook/components/nuxt-link.client.js] 6.75 KiB {main} [built]
[./.nuxt-storybook/index.js] 16.8 KiB {main} [built]
[./.nuxt-storybook/middleware.js] 47 bytes {main} [built]
[./.nuxt-storybook/mixins/fetch.client.js] 3.68 KiB {main} [built]
[./.nuxt-storybook/storybook/generated-stories-entry.js] 263 bytes {main} [built]
[./.nuxt-storybook/storybook/preview.js-generated-config-entry.js] 3.26 KiB {main} [built]
[./.nuxt-storybook/storybook/storybook-init-framework-entry.js] 213 bytes {main} [built]
[./node_modules/@nuxt/components/lib/installComponents.js] 1010 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js] 2.31 KiB {vendors~main} [built]
[./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js] 2.31 KiB {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/common/polyfills.js] 120 bytes {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/preview/globals.js] 93 bytes {vendors~main} [built]
[./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined] (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined 7.68 KiB {vendors~main} [built]
    + 1904 hidden modules

ERROR in ./.nuxt-storybook/storybook/generated-stories-entry.js
Module not found: Error: Can't resolve '..../components' in 'C:\Users\rasmu\Desktop\Netlify Projekter\netlify-cms-nuxt\.nuxt-storybook\storybook'
 @ ./.nuxt-storybook/storybook/generated-stories-entry.js 5:27-165
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt-storybook/client.js ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.nuxt-storybook/storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./.nuxt-storybook/storybook/preview.js-generated-config-entry.js ./.nuxt-storybook/storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined
Child HtmlWebpackCompiler:
                          Asset      Size               Chunks  Chunk Names
    __child-HtmlWebpackPlugin_0  6.36 KiB  HtmlWebpackPlugin_0  HtmlWebpackPlugin_0
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/@storybook/core/dist/server/templates/index.ejs] 2.15 KiB {HtmlWebpackPlugin_0} [built]

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\hiberfil.sys'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\pagefile.sys'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\swapfile.sys'
```

**Comment by farnabaz** [comment]
@Jursdotme Could you provide a reproduction sample on codesandbox?

**Comment by altryne** [comment]
This issue went away for me for a while (not sure how)  and now, after I merged a bunch of commits, removed `package-lock.json` and installed the latest versions of both nuxt and this package, it returned. 
"nuxt": "^2.14.3",
"@nuxtjs/storybook": "^2.1.0",

I installed `core-js@3 @babel/runtime-corejs3` and it seemed to solve this :/

**Comment by bissolli** [comment]
Same happened here! I've started a fresh version of nuxt with the storybook module and I faced this same issue right away.

Installing `core-js@3 @babel/runtime-corejs3` as dev-dependencies solved it.

**Comment by farnabaz** [comment]
This should fix in [`2e68e33` (#96)](https://github.com/nuxt-community/storybook/pull/96/commits/2e68e330b58bd1dc5b2bc46f31f0a2246564efd2)
